.github/workflows/test.yaml

```diff
@@ -1,77 +1,78 @@
+    branches: [develop, develop-0.1, develop-0.2]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name }}
+  cancel-in-progress: true
 
 jobs:
-  test_fmt_lint:
+  cargo_test:
+    name: cargo_test
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-
-      - name: Maximize build space
-        if: runner.os == 'Linux'
-        uses: AdityaGarg8/remove-unwanted-software@v2
+      - uses: dtolnay/rust-toolchain@stable
         with:
-          remove-dotnet: 'true'
-          remove-android: 'true'
-          remove-codeql: 'true'
-          remove-docker-images: 'true'
+          toolchain: 1.80.0
+      - uses: Swatinem/rust-cache@v2
+      - run: cargo test --no-fail-fast
 
-      - name: Install nix
-        uses: cachix/install-nix-action@v27
+  rustfmt:
+    name: rustfmt
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@stable
         with:
-          install_url: https://releases.nixos.org/nix/nix-2.24.4/install
+          toolchain: 1.80.0
+          components: rustfmt
+      - run: cargo fmt --all -- --check
 
-      - uses: cachix/cachix-action@v15
+  clippy:
+    name: clippy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@stable
         with:
-          name: holochain-ci
-
-      - name: rustfmt check
-        run: |
-          cd $GITHUB_WORKSPACE
-          nix develop .#ci --command cargo fmt --all --check
-
-      - name: clippy check
-        run: |
-          cd $GITHUB_WORKSPACE
-          nix develop .#ci --command cargo clippy --all-features --all-targets -- -D warnings
-
-      - name: unittests
-        run: |
-          cd $GITHUB_WORKSPACE
-          nix develop .#ci --command cargo test --no-fail-fast
+          toolchain: 1.80.0
+          components: clippy
+      - uses: Swatinem/rust-cache@v2
+      - run: cargo clippy -- -D warnings
 
   testbuild:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        template: [ svelte, vue, lit, react, vanilla ]
+        template: [svelte, vue, lit, react, vanilla]
     steps:
       - uses: actions/checkout@v4
 
       - name: Maximize build space
         if: runner.os == 'Linux'
         uses: AdityaGarg8/remove-unwanted-software@v2
         with:
-          remove-dotnet: 'true'
-          remove-android: 'true'
-          remove-codeql: 'true'
-          remove-docker-images: 'true'
+          remove-dotnet: "true"
+          remove-android: "true"
+          remove-codeql: "true"
+          remove-docker-images: "true"
 
       - name: Install nix
-        uses: cachix/install-nix-action@v27
+        uses: cachix/install-nix-action@v30
         with:
           install_url: https://releases.nixos.org/nix/nix-2.24.4/install
 
       - uses: cachix/cachix-action@v15
         with:
-          name: holochain-ci
+          name: holochain-scaffolding-cli
```

Comment:
No, you shouldn't try to cache everything. Set `extraPullNames` to `holochain-ci`

Reply:
This is actually wrong on Wind Tunnel. I will correct that next time I'm in there.